llvm/lib/CodeGen/ComplexDeinterleavingPass.cpp

```diff
@@ -1511,6 +1690,7 @@ bool ComplexDeinterleavingGraph::collectPotentialReductions(BasicBlock *B) {
 }
 
 void ComplexDeinterleavingGraph::identifyReductionNodes() {
+  dbgs() << "identifyReductionNodes\n";
```

Comment:
I assume this has been left over from debugging.

Reply:
Yep, I got all the ones from actual debugging, but not those that I had to add to debug a merge conflict. Removed.